mappings/net/minecraft/recipe/impl/ShieldRecipes.mapping

```diff
@@ -1,8 +1,8 @@
-CLASS none/agz
-	CLASS none/agz$a
+CLASS none/agz net/minecraft/recipe/impl/ShieldRecipes
```

Comment:
My concern with calling the main class RecipeShield is that it's not actually a Recipe. It has an init method which adds the basic shield recipe, along with the inner recipe to the registry. The inner class is actually a recipe though.